public/apps/kanban-board/js/state/store.js

```
@@ -0,0 +1,124 @@
+     */
+    subscribe(callback) {
+        this.subscribers.push(callback);
+    }
+
+    /**
+     * Notify all subscribers
+     * @param {string} event - Event type
+     * @param {Object} payload - Data payload
+     */
+    notify(event, payload) {
+        this.subscribers.forEach(callback => callback(this.state, event, payload));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)